homeassistant/components/alexa/smart_home.py

```diff
@@ -1276,6 +1354,152 @@ def async_api_previous(hass, config, request, entity):
     return api_message(request)
 
 
+def api_error_temp_range(request, temp, min_temp, max_temp, unit):
+    """Create temperature value out of range API error response.
+
+    Async friendly.
+    """
+    temp_range = {
+        'minimumValue': {
+            'value': min_temp,
+            'scale': API_TEMP_UNITS[unit],
+        },
+        'maximumValue': {
+            'value': max_temp,
+            'scale': API_TEMP_UNITS[unit],
+        },
+    }
+
+    msg = 'The requested temperature {} is out of range'.format(temp)
+    _LOGGER.info(msg)
```

Comment:
Stale debug

Reply:
I'm logging this intentionally, but it might make sense to do it from the `api_error` method.